ipu-plugin/pkg/ipuplugin/ipuplugin.go

```diff
@@ -124,6 +120,24 @@ func (s *server) Stop() {
 		//Note: Deletes bridge created in EnsureBridgeExists in  Run api.
 		s.bridgeCtlr.DeleteBridges()
 	}
+
+	log.Infof("DeletePhyPortRules, path->%s, 1->%v, 2->%v", s.p4rtbin, AccApfMacList[PHY_PORT0_INTF_INDEX], AccApfMacList[PHY_PORT1_INTF_INDEX])
+	p4rtclient.DeletePhyPortRules(s.p4rtbin, AccApfMacList[PHY_PORT0_INTF_INDEX], AccApfMacList[PHY_PORT1_INTF_INDEX])
+
+	vfMacList, err := utils.GetVfMacList()
+	if err != nil {
+		log.Errorf("Unable to reach the IMC %v", err)
+	}
+	if len(vfMacList) == 0 || (len(vfMacList) == 1 && vfMacList[0] == "") {
+		log.Errorf("No NFs initialized on the host")
```

Comment:
NF or VF? "NFs initialized on host" sounds really wrong when we are going to be supporting NFs on the ACC. Should it rather be "No NFs initialized **from** the host" ? Or probably just VFs which seems more like what the variable names imply